.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
   pgbench:
     docker:
-      - image: buildpack-deps:trusty
+      - image: buildpack-deps:stretch
```

Comment:
`trusty` refers to Ubuntu 14.04
`stretch` refers to Debian 9

While we are at it, let's bump Debian to buster, shall we?
```
Ubuntu            Debian  
20.04  focal      bullseye/ sid
19.10  eoan       buster  / sid   - 10
19.04  disco      buster  / sid
18.10  cosmic     buster  / sid
18.04  bionic     buster  / sid
17.10  artful     stretch / sid   - 9
17.04  zesty      stretch / sid
16.10  yakkety    stretch / sid
16.04  xenial     stretch / sid
15.10  wily       jessie  / sid   - 8
15.04  vivid      jessie  / sid
14.10  utopic     jessie  / sid
14.04  trusty     jessie  / sid
13.10  saucy      wheezy  / sid   - 7
13.04  raring     wheezy  / sid
12.10  quantal    wheezy  / sid
12.04  precise    wheezy  / sid
11.10  oneiric    wheezy  / sid
11.04  natty      squeeze / sid   - 6
10.10  maverick   squeeze / sid
10.04  lucid      squeeze / sid
```

Reply:
Updated the tag with buster @hanefi, thanks for the list. Also @hanefi noted that trusty has reached EOL 1 year ago